ArmPkg/Library/ArmExceptionLib/ArmExceptionLib.c

```diff
@@ -90,8 +90,8 @@ InitializeCpuExceptionHandlers (
   IN EFI_VECTOR_HANDOFF_INFO  *VectorInfo OPTIONAL
   )
 {
-  RETURN_STATUS  Status;
-  UINTN          VectorBase;
+  RETURN_STATUS         Status;
+  EFI_PHYSICAL_ADDRESS  VectorBase;

```

Comment:
Could we change VectorBase to UINT64 instead?

Reply:
Yes, this is updated to UINT64 type instead.